### ISSUE TITLE
Questions about App Academy's Curriculum 

### ISSUE THREAD

**Issue by stevenbishopengineer** [fiosman/App-Academy#1]
Hello there! I've been trying to find someone who has actually went through App Academy opens curriculum. Can you tell me how your experience has been going through the curriculum? I want to know if it is worth it.

**Comment by fiosman** [comment]
Hi there! 

Thanks for your message. Just for context, I am currently tackling the Rails module, after which there are JavaScript and React modules to complete... then building your very own full-stack web applications!

Going through the curriculum on your own is a very challenging yet extremely rewarding task. I am not sure how much research you've done but the curriculum itself teaches full-stack web development from beginning to end, and it is tiers above anything else on the internet for a variety of reasons, but I think above all: it teaches computer science fundamentals. If your goal is to gain the skills necessary to become employed in the industry, then look no further -- Sign up on a/A and begin your journey. 

I am extremely satisfied with my experience so far, mainly because i've had so much flexibility to dive deeper into certain concepts that I didn't necessarily fully understand just from reading the lectures and doing the assignments. This is luxury that in-person students do not have. The curriculum itself is no different than the in-person, it is 100% identical. Besides that, the lectures are very clear and well organized, and there is an active slack channel available to seek assistance when you are stuck. There are assignments/quizzes that you complete at the end of every section within a module and they are by far the best practice for mastery learning. A lot of folks skip some assignments and I highly discourage that, I think they should all be done sequentially because you just learn so much more. Perhaps some instructions can be vague, but then again the slack channel is there to ask questions for clarification. 

So in short, again, App Academy open is the best FREE online resource to learn programming. I know some folks who have landed jobs going through this curriculum. If you have the time, dedication and willingness to learn at a deep level, then App Academy is the best fit for you. 

Hope that was helpful, let me know if you have any other questions!

**Comment by stevenbishopengineer** [comment]
This is so exciting to hear. I've gone through random Udemy courses and I've tried a ton of other learning platforms but App Academy seems to be the most complete. A problem I had while following most courses is the challenges weren't very challenging.  From what I've read, App Academy's curriculum is VERY challenging and I love that. I'm definitely going to get started as soon as possible. 

Quick question: Are you posting your progress manually or does App Academy give you starter repos to build upon? I'm curious about that because I think it's awesome that you have been able to track your progress and keep your github active while learning.

**Comment by fiosman** [comment]
Hi! App academy does encourage you to push to github whenever you have completed a project, but it is not a requirement so there are no starter repos. I have been doing this on my own to track my progress every day.
